configure.ac

```diff
@@ -53,7 +53,7 @@ AC_ARG_WITH(nopylint,
 
 [if test "$with_nopylint" = "no"]
 [then]
-    AC_PATH_PROG([PYLINT], [python3-pylint], [no])
+    AC_PATH_PROG([PYLINT], [pylint-3], [no])
```

Comment:
But after that F25 will start failing. Because with_nopylint is set only on EL. And AFAIK the name was not changed on F25.
Personally I would propose change to spec and disable pylint check in F25 and older.